script/update-templates.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source "$( dirname "${BASH_SOURCE[0]}" )/.include"
+
+cd "$ROOT_DIR"
+aws s3 sync "$(go run ./cmd/gentemplates/main.go)" s3://exo-starter-templates
```

Comment:
So I was trying to avoid having people download the full set of starters when creating a new project. The full set of starters is about 12 megabytes which would take about 12 seconds to download on a slow internet connection (8Mbps). Downloading a compressed directory means it's at most ~600KB to download.

Happy to revert it to just use git if you think that's better.

Reply:
No, this makes sense. Thanks for explaining.